frontend/app/routes/$lang+/_public+/apply+/$id+/adult/confirmation.tsx

```diff
@@ -334,25 +339,52 @@ export default function ApplyFlowConfirm() {
         </div>
       </div>
 
-      <Button
-        className="mt-5 print:hidden"
-        size="lg"
-        variant="primary"
-        onClick={(event) => {
-          event.preventDefault();
-          window.print();
-        }}
-        data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form:Print a copy of your application bottom click"
-      >
-        {t('confirm.print-btn')}
-      </Button>
-
-      <fetcher.Form method="post" noValidate className="mt-5 flex flex-wrap items-center gap-3">
-        <input type="hidden" name="_csrf" value={csrfToken} />
-        <Button variant="primary" onClick={() => sessionStorage.removeItem('flow.state')} size="lg" className="print:hidden" data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form:Exit - Confirmation click">
-          {t('apply-adult:confirm.exit')}
-        </Button>
-      </fetcher.Form>
+      <div className="max-w-min">
```

Comment:
That `div` is not needed.

Reply:
Removing max-w-min centers the buttons, do we still want this removed?
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/3928429c-7e52-4164-a0fb-e6976a59b9df)